src/metabase/core.clj

```diff
@@ -217,16 +217,26 @@
       (log/error "Metabase Initialization FAILED: " (.getMessage e))
       (System/exit 1))))
 
+(def ^:private cmd->fn
+  {:migrate      (fn [direction]
+                   (db/migrate @db/db-connection-details (keyword direction)))
+   :load-from-h2 (fn [& [h2-connection-string-or-nil]]
+                   (db/setup-db)
```

Comment:
this is also called in the `load-from-h2` function, which means it ends up getting called twice.